index.html

```diff
@@ -122,6 +177,196 @@ <h2>Methods</h2>
+    </p>
+    <pre class="idl">
+partial dictionary RTCRtpHeaderExtensionCapability {
+  RTCRtpTransceiverDirection direction = "sendrecv";
+};
+      
+partial interface RTCRtpTransceiver {
+  void setOfferedRtpHeaderExtensions(
+sequence&lt;RTCRtpHeaderExtensionCapability&gt; headerExtensionsToOffer);
+  readonly attribute FrozenArray&lt;RTCRtpHeaderExtensionCapability&gt; headerExtensionsToOffer;
+  readonly attribute FrozenArray&lt;RTCRtpHeaderExtensionCapability&gt; headerExtensionsNegotiated;
+};
+    </pre>
+    <p>
+      Let [[\HeaderExtensionsToOffer]] be an internal slot of the RTCRtpTransceiver,
+      initialized to the platform-specific list of implemented RTP header extensions.
+      The direction
+      attribute for all extensions that are mandatory to use MUST be initialized to
+      an appropriate value other than "stopped". The direction attribute for extensions
+      that will not be offered by default in an initial offer MUST be initialized to
+      "stopped".
+    </p>
+    <p class="note">The list of header extensions that MUST/SHOULD be supported is listed in
+      [[RTCWEB-RTP]], section 5.2. The "mid" extension is mandatory to use when
+      BUNDLE is in use, per [[BUNDLE]] section 9.1.
+    </p>
+    <p>
+      Let [[\HeaderExtensionsNegotiated]] be an internal slot of the RTCRtpTransceiver,
+      initialized to an empty list.
+    </p>
+    <p>
+      In the algorithms for generating initial offers in [[RTCWEB-JSEP]] section 5.1.1,
+      replace "for each supported RTP header extension, an "a=extmap" line, as specified in
+      [[RFC5285]], section 5" " with "For each RTP header extension "e"
+      listed in [[\HeaderExtensionsToOffer]] where direction is not "stopped", an "a=extmap"
+      line, as specified in [[RFC5285]], section 5, with direction taken from "e"'s direction
+      attribute."
+    </p>
+    <p>
+      In the algorithm for generating subsequent offers in [[RTCWEB-JSEP]] section 5.2.2, replace "The
+      RTP header extensions MUST only include those that are present in the most recent answer"
+      with "For each RTP header extension listed in [[\HeaderExtensionsToOffer]] that is
+      also present in the most recent answer, and where direction is not "stopped", generate
+      an appropriate "a=extmap" line with "direction" set according to the rules of [[RFC5285]]
+      section 6, considering the direction in [[\HeaderExtensionsToOffer]] to indicate the
+      answerer's desired usage". This preserves the property that the set of extensions
+      may not grow with subsequent offers.
+    </p>
+    <p>
+      In the algorithm for generating initial answers in [[RTCWEB-JSEP]] section 5.3.1, replace "For
+      each supported RTP header extension that is present in the offer" with "For each
+      supported RTP header extension that is present in the offer and is also present in
+      [[\HeaderExtensionsToOffer]] with a direction different from "stopped"".
```

Comment:
This sentence makes it sound like while we can reject an extension by stopping it, the direction cannot be affected.

For example, if "sendrecv" is offered and I use the API to set it to "sendonly", the generated answer will still say "sendrecv".
This is not the case for transceiver directions: e.g. "sendrecv" can be offered and "sendonly" or "recvonly" can be replied.

Is this on purpose?